### DISCUSSION TITLE
Retiring SEE?

### DISCUSSION THREAD

**Discussion by jkotlinski** [jkotlinski/durexforth#419] [General]
It is something that I thought for a long time, SEE does not make good use of the bytes it consumes.
Right now, it takes like 6-700 bytes, depending how it counts.
The usefulness is questionable, since all source code is anyway readily available.

Also, it does not even work particularly well anymore.
It used to be a nice gimmick, but now "see see" produces a fair amount of junk on the screen, it is not a pleasant sight.

Would you agree if I say, it is time to retire SEE?

**Comment by burnsauce** [comment]
I do agree that it's nigh-useless on all but the simplest words, and should be moved out of the core distribution.

**Comment by Whammo** [comment]
It would be pretty cool if it were re-engineered, full-scale app that one could use to traverse the dictionary.
SEE with an optional filename. that without brings up the word at latest. Cursor through the list of it's constituent words showing their addresses, size, and etc.  select one and get a similar list of it's constituent words keeping track so as to step by step back and return to the starting point.
 But yes, comment it out.

**Comment by jkotlinski** [comment]
Done... suddenly there is plenty of free space!

**Reply by IanWitham** [reply]
@jkotlinski it's still referenced in the manual. Took me a moment to realize it must have been removed from the disk image

**Comment by jkotlinski** [comment]
SEE has actually been revived in master, and thus re-added to the (online)
manual.

I suppose durexForth v5 should be released at some point, with SEE included
on the .d64 image.
